test/integration/job/job_test.go

```diff
@@ -1694,45 +1694,45 @@ func TestJobPodReplacementPolicy(t *testing.T) {
 		"feature flag true, delete pods and verify terminating status": {
 			podReplacementPolicyEnabled: true,
 			jobSpec:                     jobSpecIndexedDefault,
-			wantTerminating:             pointer.Int32(podCount),
+			wantTerminating:             ptr.To[int32](podCount),
 			wantFailed:                  int(podCount),
 		},
 		"feature flag true, delete pods, verify terminating status and recreate upon terminating": {
 			podReplacementPolicyEnabled: true,
 			jobSpec: &batchv1.JobSpec{
-				Parallelism:          pointer.Int32Ptr(podCount),
-				Completions:          pointer.Int32Ptr(podCount),
+				Parallelism:          ptr.To(podCount),
+				Completions:          ptr.To(podCount),
 				CompletionMode:       &indexedCompletion,
 				PodReplacementPolicy: podReplacementPolicy(batchv1.TerminatingOrFailed),
 			},
-			wantTerminating: pointer.Int32(podCount),
+			wantTerminating: ptr.To[int32](podCount),
```

Comment:
Inconsistent usage here and in other places. We do not need to specify the type when the type of the variable is known.

Reply:
Thanks for catching that.

Reply:
pushed a fix for that.